spec/features/repair_request_spec.rb

```diff
@@ -278,7 +278,7 @@ def stub_property_temp_annex
 
       visit new_property_repair_request_path('00000666', status: "1", guid: '123456789')
 
-      fill_in "Problem description", with: "It's broken"
+      fill_in "description-text", with: "It's broken"
```

Comment:
Why is this necessary?

Reply:
Otherwise I get this 

`
Capybara::ElementNotFound:
       Unable to find field "Problem description" that is not disabled
`

Reply:
Strange. Looks like something is broken.

Reply:
Yes, the problem is that you specified an id to the textarea, when Rails' form helpers set it to tie it to the label. You must either use Rails-generated id or pass your change to other helpers that are related to it.

Reply:
Please, do that and revert the change to the spec, as users will interact with something labeled description, not with something have specific id.

Reply:
@cdccollins :point_up: